src/webview/mod.rs

```diff
@@ -1140,8 +1140,27 @@ pub trait WebviewExtWindows {
   /// Returns WebView2 Controller
   fn controller(&self) -> ICoreWebView2Controller;
 
-  // Changes the webview2 theme.
+  /// Changes the webview2 theme.
   fn set_theme(&self, theme: Theme);
+
+  /// Sets [the memory usage target level][1] to 'Low'. There are two levels 'Low' and 'Normal' and
+  /// the default level is 'Normal'. When the application is going inactive, setting the level to
+  /// 'Low' can significantly save the memory used by the application.
+  ///
+  /// Passing `true` to `enabled` argument means the WebView will switch the level to 'Low' and
+  /// passing `false` will switch to 'Normal'.
+  ///
+  /// The logic when application goes to inactive/active is specific to the application.
+  /// [`tao::event::WindowEvent::Focused`] event may be useful.
+  ///
+  /// Please read [the guide for WebView2][2] for more details.
+  ///
+  /// This method internally uses an API added at WebView2 v1.0.1020.30. When it is called with
+  /// WebView2 older than the version, it does nothing.
```

Comment:
Since Wry dictates the SDK version we can focus on the runtime version here 
```suggestion
  /// This method uses a WebView2 API added in Runtime version 114.0.1823.32. When it is used in
  /// an older Runtime version, it does nothing.
```